internal/connectors/db/process_result_set.go

```diff
@@ -292,13 +382,6 @@ func ReadBackupScheduleFromResultSet(res result.Result) (*types.BackupSchedule,
 
 	err := res.ScanNamed(namedValues...)
 
-	if err != nil && strings.Contains(err.Error(), "count of columns less then values") {
```

Comment:
**Unfortunately,** this does not work as well. YDB Go SDK panics instead of throwing an error, so we have to duplicate code, using different `NamedValue` setups.

Reply:
Upd: passed an additional parameter so i am able to construct the list in runtime